src/layouts/AllServices.tsx

```diff
@@ -0,0 +1,542 @@
+
+import AutomationIcon from '@patternfly/react-icons/dist/js/icons/automation-icon';
+import BellIcon from '@patternfly/react-icons/dist/js/icons/bell-icon';
+import CloudSecurityIcon from '@patternfly/react-icons/dist/js/icons/cloud-security-icon';
+import DatabaseIcon from '@patternfly/react-icons/dist/js/icons/database-icon';
+import InfrastructureIcon from '@patternfly/react-icons/dist/js/icons/infrastructure-icon';
+import UsersIcon from '@patternfly/react-icons/dist/js/icons/users-icon';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+
+import Footer from '../components/Footer/Footer.js';
+import './AllServices.scss';
+
+const AllServices = () => (
+  <div id="chrome-app-render-root">
+    <Page
+      onPageResize={null} // required to disable PF resize observer that causes re-rendring issue
+      header={
+        <Masthead className="chr-c-masthead">
+          <Header />
+        </Masthead>
+      }
+    >
+      <div className="chr-render">
+        <RedirectBanner />
+        <PageSection variant={PageSectionVariants.light} className="pf-m-fill">
+          <Stack className="chr-l-stack-allservices pf-u-background-color-100">
+            <StackItem className="sticky pf-u-background-color-100">
+              <StackItem className="pf-u-pl-lg pf-u-pb-md">
+                <Title headingLevel="h2">All Services</Title>
+              </StackItem>
+              <StackItem className="pf-u-pl-lg pf-u-pb-md-on-md">
+                <div className="pf-c-search-input">
+                  <div className="pf-c-search-input__bar">
+                    <span className="pf-c-search-input__text">
+                      <span className="pf-c-search-input__icon">
+                        <i className="fas fa-search fa-fw" aria-hidden="true"></i>
+                      </span>
+                      <input
+                        className="pf-c-search-input__text-input"
+                        type="text"
+                        placeholder="Find a service"
+                        aria-label="Find a service"
+                        disabled
+                      />
+                    </span>
+                  </div>
+                </div>
+              </StackItem>
+            </StackItem>
+            <StackItem>
+              <Gallery hasGutter>
+                <Card isPlain>
+                  <CardTitle>
+                    <FontAwesomeIcon icon={faCloudArrowUp} /> Application Services
+                  </CardTitle>
+                  <CardBody>
+                    <TextContent>
+                      <Text component={TextVariants.p}>
+                        Streamline your hybrid cloud experience, reducing the operational cost and complexity of delivering cloud-native applications.
+                      </Text>
+                      <Text component={TextVariants.p}>
+                        <Text component={TextVariants.a} href="/beta/application-services/api-designer">
```

Comment:
Let's not use `href` for internal links. This would cause a full page reload, which is not ideal. There's a chrome `appNavClick` function to navigate to a certain app. This way we don't have to keep information about beta environment and just navigate to a specific app when we chose to do so.

Reply:
The ChromeLink component would be better. But yeah, we don't generally use `a` tags for internal navigation as that will cause full page reload.

Reply:
@karelhala It seems appNavClick is deprecated ? http://front-end-docs-insights.apps.ocp4.prod.psi.redhat.com/chrome/chrome-api#appNavClick

Reply:
The `ChromeLink` component is the way to go.